INSTALL.md

```diff
@@ -7,15 +7,31 @@ We have Ubuntu packages for the latest two LTS releases. Here is how you install
 
 ### Downloading and verifying the Debian package
 
+#### Bash shell download and verification instructions
+
 The following steps will download and verify the debian package
 
+```
     sudo apt install wget gnupg
     cd /tmp
-    wget http://dist.metrist.io/orchestrator/ubuntu/ubuntu-20.04.latest.txt
-    wget http://dist.metrist.io/orchestrator/ubuntu/$(cat ubuntu-20.04.latest.txt)
-    wget http://dist.metrist.io/orchestrator/ubuntu/$(cat ubuntu-20.04.latest.txt).asc
-    wget https://github.com/Metrist-Software/orchestrator/main/dist/trustedkeys.gpg
+    wget -nc http://dist.metrist.io/orchestrator/ubuntu/ubuntu-20.04.latest.txt
+    wget -nc http://dist.metrist.io/orchestrator/ubuntu/$(cat ubuntu-20.04.latest.txt)
+    wget -nc http://dist.metrist.io/orchestrator/ubuntu/$(cat ubuntu-20.04.latest.txt).asc
+    wget -nc https://github.com/Metrist-Software/orchestrator/main/dist/trustedkeys.gpg
     gpg --keyring ./trustedkeys.gpg --verify $(cat ubuntu-20.04.latest.txt).asc
+```
+
+#### Fish shell downloand and verification instructions
```

Comment:
Fish is not a standard shell on Ubuntu (server) environments, and probably not even on desktops. Why add it?